src/CommunityToolkit.Maui/Converters/IsStringNullOrEmptyConverter.cs

```diff
@@ -7,18 +7,23 @@ namespace CommunityToolkit.Maui.Converters;
 /// <summary>
 /// Converts the incoming value to a <see cref="bool"/> indicating whether or not the value is null or empty.
 /// </summary>
-public class IsNullOrEmptyConverter : ValueConverterExtension, ICommunityToolkitValueConverter
+public class IsStringNullOrEmptyConverter : ValueConverterExtension, ICommunityToolkitValueConverter
 {
 	/// <summary>
-	/// Converts the incoming value to a <see cref="bool"/> indicating whether or not the value is null or empty.
+	/// Converts the incoming string to a <see cref="bool"/> indicating whether or not the string is null or empty using string.IsNullOrEmpty.
 	/// </summary>
 	/// <param name="value">The value to convert.</param>
 	/// <param name="targetType">The type of the binding target property. This is not implemented.</param>
 	/// <param name="parameter">Additional parameter for the converter to handle. This is not implemented.</param>
 	/// <param name="culture">The culture to use in the converter. This is not implemented.</param>
 	/// <returns>A <see cref="bool"/> indicating if the incoming value is null or empty.</returns>
 	[return: NotNull]
-	public object? Convert(object? value, Type? targetType, object? parameter, CultureInfo? culture) => ConvertInternal(value);
+	public object Convert(object? value, Type? targetType, object? parameter, CultureInfo? culture)
```

Comment:
Should return a null object to match the interface 

```suggestion
	public object? Convert(object? value, Type? targetType, object? parameter, CultureInfo? culture)
```